src/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Index Page</title>
+    <link rel="stylesheet" href="dist/main.css">
+
+</head>
+<body>
+    <div class="container">
+        <h1>LeaderBoard</h1>
+        <section class="grid">
+            <div class="grid-item">
+                <div class="d-flex justify-content-between align-items-center">
+                    <h2 class="mb-0">Recent scores</h2>
+                    <div class="ms-3">
+                        <button id="btn" class="btn btn-primary">Refresher</button>
```

Comment:
- [ ] When click on the `Refresher` button nothing happen. Kindly make sure the "Refresher" button get all scores for a game created by you from the API (receiving data from the API and parsing the JSON) as requested in the project requirement.
![Screenshot from 2023-05-25 12-25-20](https://github.com/kayjeee/leaderboard-v2/assets/91142432/85a4eadb-a8ba-467f-8961-9feaf0997768)